src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Generated.PowerShell.Commands/SwaggerPathCommands/Add-AzsScaleUnitNode.ps1

```diff
@@ -0,0 +1,144 @@
+<#
+Copyright (c) Microsoft Corporation. All rights reserved.
+Licensed under the MIT License. See License.txt in the project root for license information.
+#>
+
+<#
+.SYNOPSIS
+    
+
+.DESCRIPTION
+    Scale out a scale unit.
+
+.PARAMETER NodeList
+    A list of input data that allows for adding a set of scale unit nodes.
+
+.PARAMETER ResourceGroupName
+    Name of the resource group.
+
+.PARAMETER ScaleUnit
+    Name of the scale units.
+
+.PARAMETER Location
+    Location of the resource.
+
+#>
+function Add-AzsScaleUnitNode
+{
+    [CmdletBinding(DefaultParameterSetName='ScaleUnits_ScaleOut')]
+    param(    
+        [Parameter(Mandatory = $true, ParameterSetName = 'ScaleUnits_ScaleOut')]
+        [Microsoft.AzureStack.Management.Fabric.Admin.Models.ScaleOutScaleUnitParametersList]
```

Comment:
@deathly809 where does the user get the value of this parameter from?

Reply:
Whoops.  I guess they didn't get added.